mdx/emit.ts

```diff
@@ -86,7 +86,16 @@ const CATEGORY_LABELS = {
     promises.push(
       emitMessagesJson({
         filePath: pathToPlugin,
-        messages: pkg.messagesData,
+        messages: pkg.messagesData.concat(
+          // Concat with this array because we'll need the inner messages
+          // in the dictionary for syntax coloring.
```

Comment:
Can we add a sample of the concatenated data here? A short example will do, will help when we port this to another language, without putting console.log here.

Reply:
Sure! Would it be better if we put it here or in the `MessageData` interface?

Reply:
I think here is better, both will be perfect :). Because when porting I'll first check what these values() and map() work, and what's the outputs are.

Reply:
Got it! I have added the sample object in the comment below. PTAL!